test/Dex.test.js

```diff
@@ -15,6 +17,169 @@ contract("IntegratedLimitOrderDex", (accounts) => {
   });
 
   it("should support placing sell orders", async () => {
-    await dex.sell(100, 10);
+    const oldBalance = (await dex.balanceOf.call(accounts[0]))
+    let tx = await dex.sell(100, 10);
+    const newBalance = (await dex.balanceOf.call(accounts[0]))
+
+    assert.equal(newBalance, oldBalance - 100, "Sell order did not update balance")
+    assert.equal(tx.logs[2].event, "NewSellOrder", "Incorrect event")
+    assert.equal(tx.logs[3].event, "OrderIncrease", "Incorrect event")
+  });
+  
+  it("should support placing multiple buy orders", async () => {
+    const amount = 1;
+    const oneEtherInWei = web3.utils.toWei("1");
```

Comment:
Just use 1 wei. You also don't need to convert to a BN here; a string will automatically be converted.

Reply:
Thanks for heads up!